internal/configuration/platform/export.go

```diff
@@ -47,12 +48,37 @@ func initFormatOption() {
 	}
 }
 
+func initServiceGroupOption() {
+	cobraParamName := "service-group"
+	cobraValue := new(customtypes.ExportServiceGroup)
+	defaultValue := customtypes.ExportServiceGroup("")
+	envVar := "PINGCLI_EXPORT_SERVICE_GROUP"
+	options.PlatformExportServiceGroupOption = options.Option{
+		CobraParamName:  cobraParamName,
+		CobraParamValue: cobraValue,
+		DefaultValue:    &defaultValue,
+		EnvVar:          envVar,
+		Flag: &pflag.Flag{
+			Name:      cobraParamName,
+			Shorthand: "g",
+			Usage: fmt.Sprintf(
+				"Specifies the service group to export. "+
+					"\nOptions are: %s.",
+				strings.Join(customtypes.ExportServiceGroupValidValues(), ", "),
+			),
+			Value: cobraValue,
+		},
+		Sensitive: false,
+		Type:      options.ENUM_EXPORT_SERVICE_GROUP,
+		ViperKey:  "export.serviceGroup",
+	}
+}
+
 func initServicesOption() {
 	cobraParamName := "services"
 	cobraValue := new(customtypes.ExportServices)
-	defaultValue := customtypes.ExportServices(customtypes.ExportServicesValidValues())
+	defaultValue := customtypes.ExportServices([]string{})
```

Comment:
The (default) section in the Flag usage below needs updating since there is no more default values